todo_app/app.py

```diff
@@ -8,4 +9,11 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=get_items())
+
+
+@app.route('/items', methods=["POST"])
+def add_items():
+    add_item(request.form.get('item-title'))
+    return redirect('/')
```

Comment:
It would be better to use url_for("index") here rather than "/".

That allows you to target the function name, rather than the route part.

https://flask.palletsprojects.com/en/2.1.x/api/#flask.url_for